recipes/opencv/4.x/conanfile.py

```diff
@@ -284,6 +298,19 @@ def add_components(components):
                 if self.settings.os == "Linux":
                     self.cpp_info.components[conan_component].system_libs = ["dl", "m", "pthread", "rt"]
 
+                if self.settings.os == "Android":
+                    self.cpp_info.components[conan_component].includedirs = [
+                        os.path.join("sdk", "native", "jni", "include")]
+                    self.cpp_info.components[conan_component].system_libs.append("log")
+                    # self.cpp_info.components[conan_component].system_libs.append("mediandk")  # available for ANDROID_NATIVE_API_LEVEL > 20
```

Comment:
you can check `self.settings.os.api_level`

Reply:
Ok, I'll try to use the internal rules for this using `self.settings.os.api_level`, this looks simple enough.